apps/aevm/src/aevm_ae_primops.erl

```diff
@@ -113,13 +112,14 @@ is_local_primop(Data) ->
 %% Basic account operations.
 %% ------------------------------------------------------------------
 
-spend_call(Value, Data, State) ->
+spend_call(Value, Data, #chain{api = API, state = State} = Chain) ->
     [Recipient] = get_args([word], Data),
     %% TODO: This assumes that we are spending to an account
     RecipientId = aec_id:create(account, <<Recipient:256>>),
-    Callback = fun(API, ChainState) ->
-                       API:spend(RecipientId, Value, ChainState) end,
-    no_dynamic_cost(fun() -> cast_chain(Callback, State) end).
+    {ok, Tx} = API:spend_tx(RecipientId, Value, State),
+    Callback = fun(ChainAPI, ChainState) ->
+                       ChainAPI:spend(Tx, ChainState) end,
+    no_dynamic_gas(fun() -> cast_chain(Callback, Chain) end).
```

Comment:
If we create a new account there is actually a state cost, but it would be confusing if spends cost differently I guess?!

Reply:
Correct, for new account there should be some gas. Luca [commented on it](https://github.com/aeternity/epoch/pull/1682#discussion_r227441198).

Reply:
Now there is [ticket](https://www.pivotaltracker.com/story/show/161473793) tracking it.